entries/s/smbc.co.jp.json

```
@@ -0,0 +1,22 @@
+{
+  "Sumitomo Mitsui Banking Corporation": {
+    "domain": "smbc.co.jp",
```

A url tag seems to be needed as no website is being served on @
```suggestion
    "domain": "smbc.co.jp",
    "url:" "https://www.smbc.co.jp/",
```